src/module.c

```diff
@@ -9772,6 +9833,9 @@ void moduleRegisterCoreAPI(void) {
     REGISTER_API(ScanKey);
     REGISTER_API(CreateModuleUser);
     REGISTER_API(SetModuleUserACL);
+    REGISTER_API(ACLCheckCommandPerm);
+    REGISTER_API(ACLCheckKeyPerm);
+    REGISTER_API(ACLCheckChannelPerm);
```

Comment:
i'm not sure i like the "perm" shorthand, since the prefix already has "ACL", and "Check", maybe we can drop it?
what do you think of these changes?
```suggestion
    REGISTER_API(ACLCheckCommand);
    REGISTER_API(ACLCheckKeyName);
    REGISTER_API(ACLCheckPubsubChannel);
```